src/resources/common.yml

```diff
@@ -503,3 +503,6 @@ common:
   posterImageUrl:
     en: Preview image URL
     de: Vorschau-Bild-URL
+  dragToReorder:
+    en: Drag to reorder
+    de: Durch Bewegen neu anordnen
```

Comment:
Maybe "Mit Drag&Drop verschieben" would sound clearer in the average user's ear.

Reply:
Changed, thanks!

On that note, I am wondering if now, for the sake of consistency and having the user get used to a pattern, should we maybe re-introduce the drag icon also for sections (with tooltip as well and in the same position as for these inner panels) ?
It might even be now that some users don't know that they can drag and drop sections. That would not surprise me.

Reply:
I'm torn between the two ways: on the one hand I like that I can just drag anywhere on the header, on the other hand I'm as concerned as you that users might not spot the possibility in the first place in lieu of the missing visual indication the icon would give. I wouldn't change it right away but discuss about it. Maybe we could even ask Radu for advice.